test/models/lo_test.rb

```diff
@@ -5,4 +5,8 @@ class LoTest < ActiveSupport::TestCase
     should validate_presence_of(:description)
     should validate_uniqueness_of(:title)
   end
+
+  context 'relationships' do
+    should have_many(:introductions).class_name('Introduction')
```

Comment:
This `class_name('Introduction')` is not necessary. 
How about the dependency destroy? should be test too.